nupic/research/monitor_mixin/plot.py

```diff
@@ -22,21 +22,24 @@
 """
 Plot class used in monitor mixin framework.
 """
-
+import logging
 import matplotlib.pyplot as plt
+import matplotlib.cm as colorModel
 
 
 
 class Plot(object):
-  """
-  A plot graphed over a list of numbers.
-  """
+
+  g_log = logging.getLogger(__name__)
+
+
+
```

Comment:
nit: 2 blank lines before methods inside classes (applies here and below in several locations)